src/components/shared/translations.js

```diff
@@ -28,6 +28,9 @@ const de = {
   choose: "Auswählen",
   computer: 'Computer',
   contextualHelp: 'Kontexthilfe',
+  dialog: {
```

Comment:
I am not sure how to structure these translations but i'd tend to sorting by component if these are special cases: `confirmDialog.confirm`.

Reply:
alright, done here --> https://github.com/demos-europe/demosplan-ui/pull/1228/commits/723c95af6ed364f74004b5456fa8d177bab7ad4d